tests/test_arraymanipulation.py

```diff
@@ -605,3 +601,10 @@ def test_2D_array2(self):
     def test_generator(self):
         with assert_warns(FutureWarning):
             dpnp.vstack((numpy.arange(3) for _ in range(2)))
+
+
+@pytest.mark.parametrize("arys", [1.0, dpnp.arange(9.0).reshape(3, 3), [3, 4]])
```

Comment:
please remove `allow_fall_back_on_numpy` fixture from `tests/third_party/cupy/manipulation_tests/test_dims.py::TestDims::test_atleast_1d1` and `tests/third_party/cupy/manipulation_tests/test_dims.py::TestDims::test_atleast_1d2`